README.md

```diff
@@ -26,10 +26,15 @@ Start PayGate and its dependencies:
 docker-compose up
 ```
 
-Create an example Transfer
+Create an example Transfer with shell script
 ```
 ./examples/getting-started.sh
```

Comment:
I was hoping we delete this bash script.

Reply:
I can do that. Wasn't sure if we wanted to keep it around.